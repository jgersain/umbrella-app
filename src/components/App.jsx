import React from 'react';
import { BrowserRouter, Switch, Route } from 'react-router-dom';

import Layout from './Layout';
import Home from '../pages/Home';
import Login from '../pages/Login';
import Register from '../pages/Register';
import Looking from '../pages/Looking';
import NotFound from '../pages/NotFound';

const App = () => (
  <BrowserRouter>
    <Layout>
      <Switch>
        <Route exact path="/" component={Home} />
        <Route exact path="/login" component={Login} />
        <Route exact path="/sign_up" component={Register} />
        <Route exact path="/looking" component={Looking} />
        <Route component={NotFound} />
      </Switch>
    </Layout>
  </BrowserRouter>
);

export default App;