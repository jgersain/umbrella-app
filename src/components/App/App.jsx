import React from 'react';
import { BrowserRouter, Switch, Route } from 'react-router-dom';

import Layout from '../Layout';
import Home from '../../pages/Home';
import NotFound from '../../pages/NotFound';

const App = () => (
  <BrowserRouter>
    <Layout>
      <Switch>
        <Route exact path="/" component={ Home } />
        <Route component={ NotFound } />
      </Switch>
    </Layout>
  </BrowserRouter>
);

export default App;