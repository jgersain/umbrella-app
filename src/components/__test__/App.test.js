import React from 'react';
import App from '../App';
import renderer from 'react-test-renderer';

describe('snapshot test', () => {
  it('render correctly', () => {
    const component = renderer.create(
      <App />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
});