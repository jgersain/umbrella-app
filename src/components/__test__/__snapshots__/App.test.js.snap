// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test render correctly 1`] = `
<h1>
  This is a new shit
</h1>
`;
